code-scanning/properties/rust-clippy.properties.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "rust-clippy",
+    "creator": "The Rust Programming Language",
+    "description": "A collection of lints to catch common mistakes and improve your Rust code.",
+    "iconName": "rust-clippy",
```

Comment:
Can we use the existing rust icon as it looks quite similar?

Reply:
Updated icon name to existing rust icon.